dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -27,6 +27,7 @@ internal class RumSessionScope(
     private val parentScope: RumScope,
     internal val samplingRate: Float,
     internal val backgroundTrackingEnabled: Boolean,
+    internal val trackFrustrations: Boolean,
```

Comment:
Lol, another parameter in the Constructor :D. We really need to find a scalable solution for this @xgouchet @0xnm.

Reply:
Yes but then you will move the problem in the container (context). I guess it will be much more cleaner though as we only change it there and not in all the scopes constructor.